src/test/java/StringManipulationTest.java

```diff
@@ -87,25 +117,49 @@ public void testGeSubStrings1() {
         assertEquals(sStings[1], "string");
     }
 
+    // This tests if the method will return many words from the middle of a longer sentence
     @Test
     public void testGeSubStrings2() {
-        fail("Not yet implemented");
+        manipulatedstring.setString("This is my test string sentence that should only return a few words");
+        String [] sStings = manipulatedstring.getSubStrings(5, 9);
+        String[] expected = { "string", "sentence", "that", "should", "only" };
+        assertArrayEquals(expected, sStings);
     }
+
+    // This tests if the method will return illegal argument exception where "startWord" > "endWord"
     @Test
     public void testGeSubStrings3() {
-        fail("Not yet implemented");
+        manipulatedstring.setString("This is my test string");
+        assertThrows(IllegalArgumentException.class, () -> {
+            manipulatedstring.getSubStrings(2, 0);
+        });
     }
+
+    // This tests if the method will return illegal argument exception where "endWord" < 0
     @Test
     public void testGeSubStrings4() {
-        fail("Not yet implemented");
+        manipulatedstring.setString("This is my test string");
+        assertThrows(IllegalArgumentException.class, () -> {
```

Comment:
Is there any way to ensure that this test case is throwing the illegal argument exception in response to the negative number argument and not because the start word is greater than the end word, given that both throw the same exception type.